library/src/main/java/io/github/luizgrp/sectionedrecyclerviewadapter/Section.java

```diff
@@ -39,11 +40,13 @@
      * @param itemResourceId layout resource for its items
      * @param loadingResourceId layout resource for its loading state
      * @param failedResourceId layout resource for its failed state
+     * @param emptyResourceId layout resource for its empty state
      */
-    public Section(int itemResourceId, int loadingResourceId, int failedResourceId) {
```

Comment:
OK. I get what you mean.

So, for `SectionParameters`, I design in the way such that, resouce id for LOADED will be mandatory, and resource id for LOADING, FAILED and EMPTY will be optional. Is that fine? It will be something like ...

    SectionParameters sectionParameters = new SectionParameters.Builder(loaded_resource_id)
        .loadingResourceId(loading_resource_id)
        .emptyResourceId(empty_resource_id)
        .build();
    
or
    SectionParameters sectionParameters = new SectionParameters.Builder(loaded_resource_id)
        .build();